eva_sub_cli/validators/validator.py

```diff
@@ -544,9 +544,9 @@ def _collect_file_info_to_metadata(self):
                     file_path_2_file_size[vcf_file] = file_size
                     file_name_2_file_size[os.path.basename(vcf_file)] = file_size
         else:
-            self.error(
-                f"Cannot locate file_info.txt at {os.path.join(self.output_dir, 'other_validations', 'file_info.txt')}"
-            )
+            error_txt =  f"Cannot locate file_info.txt at {os.path.join(self.output_dir, 'other_validations', 'file_info.txt')}"
+            self.error(error_txt)
+            errors.append({'property': '/files', 'description': error_txt})
```

Comment:
I don't think this error is necessary either - if `file_info.txt` is not found, then all the file info dicts will be empty and all the errors in lines 561-569 should trigger.